src/test/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderMatchAPIIntegrationTest.kt

```diff
@@ -58,7 +56,7 @@ internal class OffenderMatchControllerAPIIntegrationTest : AbstractTestExecution
   }
 
   @Nested
-  inner class `basic operation` {
+  inner class BasicOperation {
```

Comment:
In vain hope to a fix an issue where Intellij can't jump to specific test methods when there are many inner class tests - didn't work but this just as readble